.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: ruby
+dist: trusty
 
 rvm:
-  - 2.2
+  # - "2.2"
+  - "2.3.1"
 
-branches:
-  only:
-  - master
-  - develop
-  - deploy-release
-  - /^issue.*$/
+gemfile:
+  - cf.Gemfile
+
+# branches:
```

Comment:
A couple other buildpacks still have a 2.2 version in there but I don't know of a reason it's needed at this point either.